sw/device/lib/crypto/include/datatypes.h

```diff
@@ -320,9 +322,12 @@ typedef enum otcrypto_key_mode {
   // Key is intended for KDF-CTR with HMAC as PRF.
   kOtcryptoKeyModeKdfCtrHmac =
       kOtcryptoKeyTypeKdf << 16 | kOtcryptoKdfKeyModeCtrHmac,
-  // Key is intended for KDF-CTR with KMAC as PRF.
-  kOtcryptoKeyModeKdfCtrKmac =
-      kOtcryptoKeyTypeKdf << 16 | kOtcryptoKdfKeyModeCtrKmac,
+  // Key is intended for KDF-CTR with KMAC128 as PRF.
```

Comment:
nit:
```suggestion
  // Key is intended for KDF with KMAC128 as PRF.
```
Same for two lines down.